rero_ils/modules/organisations/api.py

```diff
@@ -121,6 +90,28 @@ def get_record_by_viewcode(cls, viewcode):
 
         return result['hits']['hits'][0]['_source']
 
+    @classmethod
+    def get_record_by_user(cls, user):
+        """Return organisation associated with patron.
+
+        :param user: the user to check.
+        :return: Organisation record or None.
+        """
+        from ..patrons.api import current_patron, Patron
+        patron = Patron.get_patron_by_user(user) or current_patron
```

Comment:
you can also write:
```python
return Patron.get_patron_by_user(user) or current_patron or None
```

Reply:
I don't think so because I have to return the organisation not the patron.